conans/client/conf/__init__.py

```diff
@@ -141,7 +141,7 @@
 # no_proxy_match = *bintray.com*, https://myserver.*
 
 [hooks]    # environment CONAN_HOOKS
-attribute_checker
+attribute_checker/attribute_checker.py
```

Comment:
This will fail in an upgrade?

Reply:
I think I will leave it as-is, it is ok to have hooks also not in folders, isn't it?

Reply:
It doesn't fail, I've tried it, but eventually, you end with the `attribute_checker.py` in the root and also i the `attribute_checker/attribute_checker.py`. Maybe the Conan ones are ok in the root to simplify